pkg/runtime/apex.axdl

```diff
@@ -23,19 +23,19 @@ type ResourcesConfig {
 
 "The main Iota component/application configuration."
 type BusConfig {
-  "The Iota identifier."
+  "The Application identifier."
   id: string
-  "The Iota version."
+  "The Application version."
   version: string
   "The main code binary (wasm or native)."
-  main: string
+  main: string?
   "The interface definition."
-  spec: string
+  spec: string?
   "Package specifies the contents of the Iota OCI image."
   package: Package?
   "Resources are externally configured sources and receivers of data (DB, REST endpoint)."
   resources: [string]?
-  "Other Iotas that this Iota depends on using."
+  "Imported Iota dependencies."
```

Comment:
As I mentioned above, perhaps we use a different term here (e.g. `uses` or `imports`). `uses` is my preference... I guess. :)

Reply:
On second thought, I'm using `uses` for something else in the TypeScript stuff (e.g. `app.uses(RestModuleV1(...))`). Maybe we want `imports`.